.yamllint

```diff
@@ -0,0 +1,3 @@
+---
+rules:
+  line-length: disable
```

Comment:
:(

What would be needed to enable that check again?

Reply:
Adjusted to a max of 140